### PR TITLE
Edited README.md

### PR DESCRIPTION
**Pull request by MasaTate**
I edeited README.md.
Please check it out !

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# test_pull_requests
\ No newline at end of file
+# test_pull_requests
+
+Hi! I'm Masatoshi Tateno.
+I'm interested in Computer Vision.
```
Comment (kenshiro-sato) [inline README.md:4]: Thank you for your introduction!
Nice to meet you!
What kind of food do you like?

**Review by kenshiro-sato — APPROVED** [review]
Thank you! just one comment

**Review by kenshiro-sato — CHANGES_REQUESTED** [review]
Thank you! just one comment

**Comment by MasaTate** [comment]
Thank you for your review !
I edited `README.md` again.
Please check it out.

File: README.md [hunk 2]
```diff
@@ -2,3 +2,5 @@
 
 Hi! I'm Masatoshi Tateno.
 I'm interested in Computer Vision.
+
+I like any spicy food. For example, Mabo tofu.
```
Comment (kenshiro-sato) [inline README.md:6]: We are friends, MasaTate San. I love Mabo tofu, too.

**Review by kenshiro-sato — APPROVED** [review]

**Comment by kenshiro-sato** [comment]
Thank you again!
I approved your PR!!

**Pull request merged**